paddle/fluid/framework/ir/mkldnn/params_quantization_mkldnn_pass.cc

```diff
@@ -151,16 +134,34 @@ void ParamsQuantizationMkldnnPass::QuantizeConv(ir::Graph* graph,
     PADDLE_ENFORCE_NOT_NULL(
         scope, platform::errors::InvalidArgument("Scope cannot be nullptr."));
 
-    if (ShouldSkipConv(conv_op, scope, conv_filter)) {
+    // If not a quantized OP
+    if (!platform::HasOpINT8DataType(conv_op->Op())) {
       return;
     }
 
-    QuantizeConvInput<int8_t>(
-        scope, g, conv_op, conv_filter->Name(), "Scale_weights");
+    auto filter_var = scope->GetVar(conv_filter->Name());
+    if (filter_var->Get<LoDTensor>().dtype() != phi::DataType::FLOAT32) {
+      VLOG(0) << "Skipping convolution filter: " << conv_filter->Name()
```

Comment:
I think you can easily move these new checks to the function above QuantizeConvInput to avoid duplicating the code